### PR TITLE
Host specification in repo instead of Google Docs

### PR DESCRIPTION
**Pull request by paulmillr**
There is zero need to trust Google for hosting. 16 people agreed with me in the corresponding discussion: https://github.com/FiloSottile/age/discussions/228

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
+## A simple file encryption tool & format
+
+> Filippo Valsorda (@FiloSottile) — Ben Cartwright-Cox (@Benjojo12)
+
+> Designed at the  [Recurse Center](https://recurse.com/) during NGW 2019
```
Comment (waldyrious) [inline spec.md:5]: Nit: double space

```suggestion
> Designed at the [Recurse Center](https://recurse.com/) during NGW 2019
```

**Comment by waldyrious** [comment]
Heads up: space characters appear to have been added around links (perhaps as an artifact of the conversion?), which leads to double spaces in various places throughout the doc.

**Comment by FiloSottile** [comment]
Thank you for the PR. I've extracted and polished the most relevant bits of the spec and published it at https://c2sp.org/age, where it will be maintained and versioned separately from the Go implementation.

**Comment by paulmillr** [comment]
Great, finally, thanks

**Pull request closed without merging**